content/applications/hr/payroll/reporting.rst

```diff
@@ -2,68 +2,103 @@
 Reporting
 =========
 
-The :guilabel:`Reporting` section of the *Payroll* app offers a variety of reports to choose from.
-The :guilabel:`Payroll`, :guilabel:`Meal Vouchers`, :guilabel:`Attachment of Salary`, and
-:guilabel:`DMFA` reports are universal and available for all companies regardless of location. The
-:guilabel:`Paid Time Off Allocation`, :guilabel:`273S Sheet`, and :guilabel:`274.XX Sheets` reports
-are specific to Belgian companies only.
+The :guilabel:`Reporting` section of the *Payroll* app offers a variety of reports to choose from,
+organized by location.
 
-To view a report, go to :menuselection:`Payroll --> Reporting` and click on the specific report.
+The :guilabel:`Payroll` report, :guilabel:`Work Entry Analysis` report, and :guilabel:`Salary
+Attachment Report` are default reports in the *Payroll* app, and are available for all companies,
+regardless of location.
+
+Beneath the three default reports are all localization-based reports, organized by country, in
+alphabetical order. These reports contain all the various information for the offered benefits and
+local tax laws.
+
+To view all the available reports for the database, including all the localization-specific ones,
+navigate to :menuselection:`Payroll app --> Reporting` to view the available reports. Click on a
+specific report to view it.
 
 .. image:: reporting/reports.png
    :align: center
    :alt: Report dashboard view.
 
-If a report is unavailable to a company, a user error pops up, stating `You must be logged in to a
-Belgian company to use this feature.`
+If a report is unavailable to a user, an :guilabel:`Invalid Operation` pop-up window appears,
+stating: `You must be logged in to a (country) company to use this feature.`
+
+Filters
+=======
+
+At the top of each report, the default filters are shown inside the :guilabel:`Search` box.
 
-At the top of each report, click on :guilabel:`Filters` to display the optional filters for the
-specific report. Filters show information that matches the specific filter parameters. For example,
-when selecting the :guilabel:`Last 365 Day Payslip` filter, only payslips for the last 365 days
-appear. All reports have the option to add a custom filter, and each have different default custom
-filter options. Select the parameters, then click the :guilabel:`Apply` button.
+Click the :guilabel:`drop-down arrow` in the search box to display the available filters. Filters
+show information that matches the specific filter parameters.
+
+.. example::
```

Comment:
I would use a guilabel for the filters, as you aren't manually entering the info, you are selecting them from options on the UI